lib/situation_room/sites.ex

```diff
@@ -25,17 +25,31 @@ defmodule SituationRoom.Sites do
     %Site{}
     |> Site.changeset(params)
     |> Repo.insert()
+    |> broadcast(:site_created)
   end
 
   # Delete a site from the database by specifying specific field
   # param ex: (name: "mojotech") or (endpoint: "http://mojo.com")
   def delete_site(site) do
     Repo.delete(site)
+    |> broadcast(:site_deleted)
   end
 
   def update_site(%Site{} = site, attrs) do
     site
     |> Site.changeset(attrs)
     |> Repo.update()
+    |> broadcast(:site_updated)
+  end
+
+  def subscribe do
+    Phoenix.PubSub.subscribe(SituationRoom.PubSub, "sites")
```

Comment:
Can we move these to a module attribute `@topic "sites"`

Reply:
Done 👍